std/sys/Http.hx

```diff
@@ -58,12 +58,14 @@ class Http extends haxe.http.HttpBase {
 			onError(e);
 		}
 		customRequest(post,output);
-		if(!err)
+		if(!err){
+			onBytes(output.getBytes());
```

Comment:
I think we can't call `getBytes()` here twice (see docs on https://api.haxe.org/haxe/io/BytesOutput.html#getBytes)

Best to store it in a variable before passing to onBytes and onData

Reply:
(the reason being is that this works on some platforms but fails on others)